backend/spec/features/admin/products/edit/images_spec.rb

```diff
@@ -64,6 +64,8 @@
           expect(page).to have_xpath("//img[contains(@src,'ror_ringer')]")
         end
       end
+
+      expect(Spree::Image.last.viewable).to eq(product.master)
```

Comment:
Not easily. The regression here was that the value wasn't being sent to the server and not saved in the DB. I think this is the simplest test for that.